### PR TITLE
[Modular] Donator stuff

### PR DESCRIPTION
**Pull request by StrangeWeirdKitten**
## About The Pull Request

Adds mine and Synthax's donator item.

Mine is another ring/hypno that flashes fancy text on a whisper, and Synthax is a bunny plush of their spritework.
Wehweh told me the idea sounded good but to check with the maintainers. So here's the PR.

image

## Proof of Testing

<details>
  
image
https://i.gyazo.com/bcce4515a148655ce0f35729ae6b60e0.gif

</details>

### CODE REVIEW

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -572,3 +572,12 @@
...
 	squeak_override = list('modular_skyrat/modules/customization/game/objects/items/sound/merowr.ogg' = 1)
+
+//Donation reward for syntax1112
+/obj/item/toy/plush/skyrat/plushie_syntax1112
+	name = "Lop Bunny Plushie"
+	desc = "A floppy-eared rabbit in marketable plushie form. Complete with an internal self-inflating squeaker device!"
+	icon_state = "fuzz_bunny"
+	attack_verb_continuous = list("nibbles", "squeaks", "nose twitches", "thumps", "falls over")
+	attack_verb_simple = list("nibble", "squeak", "nose twitch", "thump", "fall over")
+	squeak_override = list('modular_skyrat/modules/customization/game/objects/items/sound/deerplush.ogg')
```
Comment (Zonespace27) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]:
```suggestion
	squeak_override = list('modular_skyrat/modules/customization/game/objects/items/sound/deerplush.ogg' = 1)
```

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm [hunk 2]
```diff
@@ -346,3 +346,4 @@
 	else if(arousal > AROUSAL_MINIMUM_DETECTABLE)
 		. += span_purple("[p_they()] [p_are()] slightly blushed.") + "\n"
 
+
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm]: remove

File: tgui/packages/tgui-panel/styles/goon/chat-dark.scss [hunk 3]
```diff
@@ -841,6 +841,30 @@ em {
     text-shadow: 0 0 4px #c51e1e;
   }
 }
+
+.glossy {
```
Comment (Zonespace27) [inline tgui/packages/tgui-panel/styles/goon/chat-dark.scss:845]: add to lightmode too

File: code/__DEFINES/~skyrat_defines/lewd_defines.dm [hunk 4]
```diff
@@ -17,6 +17,7 @@
 #define TRAIT_BIMBO "bimbo"
 #define TRAIT_RIGGER "rigger"
 #define TRAIT_ROPEBUNNY "rope bunny"
+#define TRAIT_HYPNOTIC "hypnotic"
```
Comment (Zonespace27) [inline code/__DEFINES/~skyrat_defines/lewd_defines.dm]: why

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 5]
```diff
@@ -1365,3 +1365,12 @@
 	icon_state = "long-coat"
 	hoodtype = /obj/item/clothing/head/hooded/sigmarcoat
 	supports_variations_flags = NONE
+
+// Donation reward for Sonicgotnuked
+
+/obj/item/clothing/gloves/ring/hypno/nuke
+	name = "Suspiciously Glossy Ring"
```
Comment (Zonespace27) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]: lower case

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 6]
```diff
@@ -572,3 +572,12 @@
 	attack_verb_continuous = list("snuggles", "nibbles", "awoos", "tail whaps")
 	attack_verb_simple = list("snuggle", "nibble", "awoo", "tail whap")
 	squeak_override = list('modular_skyrat/modules/customization/game/objects/items/sound/merowr.ogg' = 1)
+
+//Donation reward for syntax1112
+/obj/item/toy/plush/skyrat/plushie_syntax1112
+	name = "Lop Bunny Plushie"
```
Comment (Zonespace27) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]: lower case

**Review by Zonespace27 — DISMISSED** [review]

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 7]
```diff
@@ -572,3 +572,12 @@
...
 	attack_verb_simple = list("snuggle", "nibble", "awoo", "tail whap")
 	squeak_override = list('modular_skyrat/modules/customization/game/objects/items/sound/merowr.ogg' = 1)
+
+//Donation reward for syntax1112
+/obj/item/toy/plush/skyrat/plushie_syntax1112
+	name = "lop Bunny plushie"
+	desc = "A floppy-eared rabbit in marketable plushie form. Complete with an internal self-inflating squeaker device!"
+	icon_state = "fuzz_bunny"
+	attack_verb_continuous = list("nibbles", "squeaks", "nose twitches", "thumps", "falls over")
+	attack_verb_simple = list("nibble", "squeak", "nose twitch", "thump", "fall over")
```
Comment (tf-4) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]: "falls over" doesn't really make much sense here, doubly so for its simple form

**Review by tf-4 — CHANGES_REQUESTED** [review]

**Comment by StrangeWeirdKitten** [comment]
the pains of putting crap on your master branch and they're not even merged yet.

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 8]
```diff
@@ -572,3 +572,12 @@
...
 	attack_verb_simple = list("snuggle", "nibble", "awoo", "tail whap")
 	squeak_override = list('modular_skyrat/modules/customization/game/objects/items/sound/merowr.ogg' = 1)
+
+//Donation reward for syntax1112
+/obj/item/toy/plush/skyrat/plushie_syntax1112
+	name = "lop Bunny plushie"
+	desc = "A floppy-eared rabbit in marketable plushie form. Complete with an internal self-inflating squeaker device!"
+	icon_state = "fuzz_bunny"
+	attack_verb_continuous = list("nibbles", "squeaks", "nose twitches", "thumps", "flops over")
+	attack_verb_simple = list("nibble", "squeak", "nose twitch", "thump", "flop over")
```
Comment (tf-4) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]: It still doesn't make much sense. `"You flop over John Syndicate in the chest!"`?

**Review by tf-4 — CHANGES_REQUESTED** [review]
Sorry for the delay on this one.

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 9]
```diff
@@ -572,3 +572,12 @@
 	attack_verb_continuous = list("snuggles", "nibbles", "awoos", "tail whaps")
 	attack_verb_simple = list("snuggle", "nibble", "awoo", "tail whap")
 	squeak_override = list('modular_skyrat/modules/customization/game/objects/items/sound/merowr.ogg' = 1)
+
+//Donation reward for syntax1112
+/obj/item/toy/plush/skyrat/plushie_syntax1112
+	name = "lop Bunny plushie"
```
Comment (tf-4) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]:
```suggestion
	name = "lop bunny plushie"
```

**Review by tf-4 — APPROVED** [review]

**Pull request merged**